content/en/continuous_integration/tests/_index.md

```diff
@@ -33,6 +33,24 @@ cascade:
 
 [Test Visibility][1] provides a test-first view into your CI health by displaying important metrics and results from your tests. It can help you investigate performance problems and test failures that concern you the most because you work on the related code, not because you maintain the pipelines they are run in.
 
+### Supported features
+|                                         | .NET | Java | Javascript | Python | Ruby | Swift | JUnit Xml |
+|-----------------------------------------|------|------|------------|--------|------|-------|-----------|
+| Accurate time/durations results         | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Structured stack traces                 | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Distributed traces on Integration tests | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Agent based reporting                   | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Agentless reporting                     | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
+| Test suite level visibility             | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
+| Manual API                              | ✓    | ✓    | ✓          | ✗      | ✗    | ✓     | ✗         |
+| Codeowner by test                       | ✓    | ✓    | ✓          |        |      | ✓     | ✗         |
+| Source code start/end reporting         | ✓    | ✓    | ✓          |        |      | ✓     | ✗         |
```

Comment:
I don't know if this suggestion breaks your formatting but I'd say it's important to note 
```suggestion
| Source code start/end reporting         | ✓    | ✓    | ✓ (only start)          |        |      | ✓     | ✗         |
```

Reply:
done in https://github.com/DataDog/documentation/pull/19793/commits/2cc0d88ca951ceee6c73960eb3fce06a6f7d00bf